WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java

```diff
@@ -157,25 +172,71 @@ void setOnSelectedItemsChangeListener(OnSelectedItemsChangeListener listener) {
         return holder;
     }
 
+    private String getFormattedTitle(CommentModel comment) {
```

Comment:
I wonder if this and other methods below imported from the deleted `Comment.java` would be better as utility methods in `CommentUtils` rather than here in the `CommentAdapter`, in case we want to use them elsewhere in the future.